build.gradle

```diff
@@ -32,10 +32,10 @@ sourceSets {
 }
 
 dependencies {
-    implementation group: 'com.couchbase.client', name: 'scala-client_2.12', version: '1.7.5'
-    implementation group: 'com.couchbase.client', name: 'tracing-opentelemetry', version: '1.5.5'
-    implementation group: 'com.couchbase.client', name: 'metrics-opentelemetry', version: '0.7.5'
-    implementation group: 'com.couchbase.client', name: 'metrics-micrometer', version: '0.7.5'
+    implementation group: 'com.couchbase.client', name: 'scala-client_2.12', version: '1.7.6'
```

Comment:
It's great that you remember build.gradle - but don't forget https://github.com/mikereiche/docs-sdk-scala/blob/release/1.7/modules/devguide/examples/scala/pom.xml !

It's in the README:  https://github.com/mikereiche/docs-sdk-scala/tree/release/1.7?tab=readme-ov-file#patch-releases